.rubocop.yml

```diff
@@ -10,21 +13,21 @@ AllCops:
 Metrics/ClassLength:
   Max: 330
 
-Encoding:
-  Exclude:
-    - lib/rspec/core/formatters/exception_presenter.rb
+# This can be addressed once Ruby 1.8 support is removed.
+Style/Encoding:
+  Enabled: false
 
 # This should go down over time.
 Metrics/AbcSize:
-  Max: 40
+  Max: 283
 
 # This should go down over time.
-Metrics/LineLength:
-  Max: 130
+Layout/LineLength:
+  Max: 172
 
 # This should go down over time.
 Metrics/MethodLength:
-  Max: 39
+  Max: 134
```

Comment:
Why the increases? If its the few rule exception removals we'd rather have a lower overall limit and have the exceptions.

Reply:
@JonRowe 

So I wasn't really seeing a few rule exceptions - there were a large number of violations of each of these.  That said, this is something that can be addressed by individual exceptions if that's desired.

Reply:
I think we could have our target settings in `.rubocop.yml` and temporary allowances in `.rubocop_todo.yml`.

We surely don't aim for methods with 134 lines.

Reply:
@pirj Because of how the include works you can't do that.  The `.rubocop_todo.yml` is included first, and the value in `.rubocop.yml` is then processed.  It will "shadow" a value specified in the `.rubocop_todo.yml`

Reply:
That's quite unfortunate, I remembered it should be the other way around.
Might be just move those over to todo file completely?

Reply:
I'm just curious why the large increase

Reply:
I'll dig into some of these after I get the Ruby 3.1/Rails 6.1 passing on rspec-rails

Reply:
So I think it's because rubocop was only run against the `lib` subdirectory, and not in a more global way.  For example:

`spec/rspec/core/filterable_item_repository_spec.rb` contains a method with AbcSize of 282.  Next highest is 147
`spec/rspec/core/filterable_item_repository_spec.rb` contains a method with 134 lines.  The next longest is 46

And there are many, many lines in the `spec` subdirectory that exceed the Layout/LineLength value of 130.

Let me tweak this and see if I can put back the old values or something close to them.

Reply:
@JonRowe Take a look at the latest changes and let me know if that looks better to you.